### ISSUE TITLE
Geant3 needs "-fallow-invalid-boz" when using gfortran 10.2

### ISSUE THREAD

**Issue by MohammadAlTurany** [FairRootGroup/FairSoft#295]
On MacOS 10.15.6 and fortran installed by brew (Homebrew GCC 10.2.0) I get the error ()

```
/Users/turany/development/FairSoft/legacy/transport/geant3/minicern/lnxgs/ublow.F:22:20:
   22 |       data iblan1/x'20202000'/
      |                    1
Error: Hexadecimal constant at (1) uses nonstandard X instead of Z [see '-fno-allow-invalid-boz']
/Users/turany/development/FairSoft/legacy/transport/geant3/minicern/lnxgs/ublow.F:23:19:
   23 |       data mask1/x'000000ff'/
      |                   1
Error: Hexadecimal constant at (1) uses nonstandard X instead of Z [see '-fno-allow-invalid-boz']
```

This can be solved by adding the flag **-fallow-invalid-boz** to the fortran flags

**Comment by dennisklein** [comment]
for reference: https://gcc.gnu.org/onlinedocs/gfortran/Fortran-Dialect-Options.html

**Comment by dennisklein** [comment]
resolved in `dev` and also reported and resolved upstream via vmc-project/geant3#17.
